Makefile

```diff
@@ -59,3 +62,8 @@ run-dev-compose:
 
 stop-dev-compose:
 	docker-compose down --volumes
+
+run-tests: | build-tests
```

Comment:
This command is already defined in line 20.

Reply:
In the next version this should be fixed.